org.eclipse.scout.rt.mom.api/src/main/java/org/eclipse/scout/rt/mom/api/ISubscription.java

```diff
@@ -35,4 +50,15 @@ public interface ISubscription extends IDisposable {
    */
   @Override
   void dispose();
+
+  /**
+   * @return true if subscription is disposing or was disposed
+   */
+  boolean isDisposed();
+
+  /**
+   * @return the stats of this subscription since the session was started
```

Comment:
add comment: "may be null when no session/connection was opened or in case of null transport"